rpc_jobs/rpc_upgrades.yml

```diff
@@ -248,6 +248,26 @@
       - 'PM_{repo_name}-{branch}-{image}-{scenario}-{action}'
     CRON: "{CRON_WEEKLY}"
 
+- project:
+    name: "rpc-upgrades-aio-newton-snapshot-incremental"
+    repo_name: "rpc-upgrades"
+    repo_url: "https://github.com/rcbops/rpc-upgrades"
+    image:
+      - xenial_snapshot:
+          REGIONS: "DFW"
+          BOOT_TIMEOUT: 1500
+    scenario:
+      - newton:
+          IMAGE: "rpc-r14-xenial_loose_artifacts-swift"
```

Comment:
`rpc-r14-xenial_loose_artifacts-swift` is not an available IMAGE, however it is an available SLAVE_TYPE. Using this will make use of nodepool, which is good - just ensure that the right key word is used.